### PR TITLE
Add epoch number in logs

### PR DESCRIPTION
**Pull request by Meylina**
Correspond à l'issue #192

### CODE REVIEW

**Review by bowni — APPROVED** [review]

**Comment by bowni** [comment]
Well done and thanks @Meylina, seems good to me. Maybe @arthurPignet or @RomainGoussault for a second review before merging?

**Review by arthurPignet — APPROVED** [review]

**Pull request merged**
